python/ray/serve/task_processor.py

```diff
@@ -0,0 +1,231 @@
+import logging
+import threading
+import time
+from abc import ABC, abstractmethod
+from typing import Any, Dict, Optional
+
+from celery import Celery
+
+from ray.serve._private.constants import SERVE_LOGGER_NAME
+from ray.serve.schema import (
+    CeleryAdapterConfig,
+    TaskProcessorConfig,
+    TaskResult,
+)
+
+logger = logging.getLogger(SERVE_LOGGER_NAME)
+
+
+class TaskProcessorAdapter(ABC):
```

Comment:
are these public apis?

Reply:
hm why is lint passing? do you not need to decorate these with api decorators indicating whether they're stable/beta/alpha?